.github/workflows/tag.yml

```diff
@@ -0,0 +1,18 @@
+name: Tag
+
+on:
+  pull_request:
+    types: closed
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Tag
+        uses: K-Phoen/semver-release-action@master
+        with:
+          release_branch: master
+        env:
+          # This is a personal access token so that we can trigger additional jobs
+          # https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
Would it be possible to rename this to `PAC_TOKEN` or something similar to avoid confusion with `GITHUB_TOKEN`.